app/src/main/java/com/palone/planahead/screens/taskEdit/ui/segmentedButton/SegmentedRadioButtonItem.kt

```diff
@@ -34,7 +36,11 @@ fun SegmentedRadioButtonItem(
             bottomEnd = roundedCornerEnd
         ), onClick = { onValueChange(label) }) {
         if (isChecked)
```

Comment:
Always use curly braces. This code style is unreadable and leads to bugs. For example: https://lwn.net/Articles/588369/